src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_name!(name)
```

Comment:
Looks like is bad naming... I mean it is used to check a valid issue name (id at the new line 76 of models/patchinfo.rb) not a valid issue tracker name... :open_mouth: